releasenotes/notes/0.1/migrate-algrithms-from-terra-6a8aa994d1206806.yaml

```diff
@@ -19,5 +19,5 @@ prelude: |
         any deprecated algorithm function.
 
     The decision to migrate the :mod:`qiskit.algorithms` module to a
-    separate package was made to clarify the purpose of `Qiskit <https:\\qiskit.org>`_ and
+    separate package was made to clarify the purpose of `Qiskit <https://www.ibm.com/quantum/qiskit>`__ and
```

Comment:
See the comment I add that included a comment around changing release notes. My thought would be to revert this.

Reply:
Oh hm. The tricky thing is that the original URL is broken because it uses `\\` rather than `//`. Any ideas what we should do, like if we can adjust the release notes logic? I'm fine reverting this if you prefer.

Reply:
Yeah, I noticed the `\\`  in your change - I saw from what is published that it does end up making a link out of it, of sorts, ends up with a 404 is you click it.

The script that checks the notes - I know we messed around with it for some time so as we could publish the docs at any time and not pull in notes that had not yet been released.

For me the easiest thing, assuming we want to have all the other changes published now, ie backport and deploy the docs again, would be to revert the change. What I think would be doable is to fix this (and any others) in main branch which I think will be ok - i.e. a separate PR not backported. The intent was to have a release not too long after Qiskit 1.0 comes out so it would be part of that and get updated when the docs are deployed for that.